bigbluebutton-html5/imports/ui/components/nav-bar/component.jsx

```diff
@@ -111,12 +110,10 @@ class NavBar extends Component {
               aria-label={ariaLabel}
               icon="user"
               className={cx(toggleBtnClasses)}
-              aria-expanded={isExpanded}
```

Comment:
This will remove the screen reader indicator to tell if the user list is open or not. Is there anything that can be done inside the tooltip component itself?